modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/metadata/ColocationGroup.java

```diff
@@ -228,6 +228,27 @@ public ColocationGroup finalaze() {
         return forNodes0(nodeIds);
     }
 
+    /** */
+    public ColocationGroup filterForPartitions(int[] parts) {
```

Comment:
Why can't we use the same naming for ColocationGroup, Fragment and FragmentMapping? (`filterForPartitions`, `filterByPartitions`)